insanely_fast_whisper_rocm/cli/commands.py

```
@@ -68,16 +133,35 @@ def translate(audio_file: Path, **kwargs) -> None:
+    identical_count = sum(
+        1 for seg in segments if (seg.get("start"), seg.get("end")) == first_timestamp
+    )
+
+    return (identical_count / len(segments)) > 0.5
 
-    Given `task` ("transcribe" or "translate") and an audio file path, perform speech recognition or translation, handle video-to-audio extraction when needed, optionally stabilize timestamps, export formatted outputs and optional benchmark data, and clean up any temporary files. On fatal runtime errors the function exits the process with a non-zero status.
 
-    Parameters:
-        task (str): The task to perform; must be either "transcribe" or "translate".
-        audio_file (Path): Path to the input audio or video file. If a supported video format is provided, its audio will be extracted and processed.
+def _run_task(*, task: str, audio_file: Path, **kwargs: dict) -> None:  # noqa: C901
```

Pylint (4.0.4)

[refactor] Too many local variables (49/15)

(R0914)